src/Microsoft.AspNetCore.Hosting/Internal/HostingLoggerExtensions.cs

```diff
@@ -126,7 +127,11 @@ public override string ToString()
             {
                 if (_cachedToString == null)
                 {
-                    _cachedToString = $"RequestId:{_httpContext.TraceIdentifier} RequestPath:{_httpContext.Request.Path}";
+                    _cachedToString = string.Format(
```

Comment:
I suppose this doesn't need to be changed? (initially thought TraceIdentifier was a number but its not)